src/main/java/nextstep/ladder/domain/LadderNames.java

```diff
@@ -0,0 +1,63 @@
+package nextstep.ladder.domain;
+
+import nextstep.ladder.exception.InputNullException;
+import nextstep.ladder.exception.StringLengthException;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+public class LadderNames {
+    private static final int LETTER_LIMIT = 5;
+
+    private List<LadderName> ladderNames;
+
+    public static LadderNames make(String ladderName) {
+        return new LadderNames(ladderName);
+    }
+
+    private LadderNames(final String ladderNames) {
+        inputValidation(ladderNames);
+        List<LadderName> list = new ArrayList<>();
+        Arrays.stream(ladderNames.split(","))
+                .map(this::lengthValidation)
+                .forEach(name -> list.add(new LadderName(name)));
+        this.ladderNames = list;
+    }
+
+    private void inputValidation(String ladderNames) {
+        if (Objects.isNull(ladderNames) || ladderNames.equals("")) {
+            throw new InputNullException();
+        }
+    }
+
+    private String lengthValidation(String name) {
+        if(name.length() > LETTER_LIMIT) {
+            throw new StringLengthException();
+        }
+        return name;
+    }
+
+    public int size() {
+        return this.ladderNames.size();
+    }
```

Comment:
```suggestion
    public int size() {
        return ladderNames.size();
    }    
```
this는 생략해도 괜찮을 것 같아요.